src/components/ColorPicker/index.tsx

```diff
@@ -27,6 +27,8 @@ const ColorPicker = ({
   const colorRef = useRef<HTMLDivElement>(null);
   const [coordinates, setCoordinates] = useState({ x: 0, y: 0 });
 
+  console.log('기능 추가'); // 100줄
```

Comment:
```suggestion
  console.log('add feature'); // 100줄
```
- 한글보다는 영어로 작성하는게 좋을 것 같습니다!

Reply:
동의합니다. (테스트 pr입니다.)  LGTM